tests/test_coverage.py

```diff
@@ -161,6 +161,18 @@ def test_process_item_creates_license_pool(self):
         eq_(True, isinstance(lp, LicensePool))
         eq_(lp.data_source, self.coverage_provider.output_source)
 
+        # Prepare an identifier that already has a LicensePool through
+        # another source.
+        licensed = self._identifier(identifier_type=Identifier.OVERDRIVE_ID)
+        other_source = DataSource.lookup(self._db, DataSource.OVERDRIVE)
+        lp = LicensePool.for_foreign_id(
+            self._db, other_source, licensed.type, licensed.identifier
+        )[0]
+
+        self.coverage_provider.process_item(licensed)
+        eq_(lp, licensed.licensed_through)
```

Comment:
I don't understand what this is testing. Are you verifying that `licensed_through` didn't get overwritten by some other value?